docs/root/install/install.rst

```diff
@@ -8,5 +8,5 @@ Building and installation
 
   building
   ref_configs
-  sandboxes/sandboxes.rst
+  getting_started
```

Comment:
I think my preference would be to move getting started before ref_configs. I think it flows better. Thoughts?